src/fidgets/community/nouns-dao/NounishGovernance.tsx

```diff
@@ -1,14 +1,19 @@
-import React, { useState } from "react";
-import { FidgetArgs, FidgetProperties, FidgetModule } from "@/common/fidgets";
-import type { ProposalData } from "@/fidgets/community/nouns-dao";
-import useGraphqlQuery from "@/common/lib/hooks/useGraphqlQuery";
-import ProposalListView from "@/fidgets/community/nouns-dao/components/ProposalListView";
-import ProposalDetailView from "@/fidgets/community/nouns-dao/components/ProposalDetailView";
 import { CardContent } from "@/common/components/atoms/card";
 import TextInput from "@/common/components/molecules/TextInput";
+import { FidgetArgs, FidgetModule, FidgetProperties } from "@/common/fidgets";
```

Comment:
Once again, you clean this up to make the changes that are being made more obvious. By changing the import order, you obfuscate what changes are being made

Reply:
same as this: https://github.com/Nounspace/nounspace.ts/pull/176/files#r1661673126